build.gradle

```diff
@@ -1,23 +1,12 @@
-buildscript {
-    repositories {
-        jcenter()
-    }
-
-    dependencies {
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
-        classpath 'org.ajoberstar:gradle-git:1.4.2'
-    }
+plugins {
+    id "groovy"
+    id "java-gradle-plugin"
+    id "com.jfrog.bintray" version "1.8.4"
+    id "maven-publish"
+    id 'org.ajoberstar.git-publish' version "2.0.0"
 }
 
-apply plugin: 'groovy'
-apply plugin: 'maven-publish'
-apply plugin: 'com.jfrog.bintray'
-apply plugin: 'org.ajoberstar.github-pages'
-apply plugin: 'java-gradle-plugin'
-
-def compatibilityVersion = 1.6
-sourceCompatibility = compatibilityVersion
-targetCompatibility = compatibilityVersion
+sourceCompatibility = "1.6"
```

Comment:
What's the reason for removing the target compatibility?

Reply:
Target is equal to source, unless you override it. So there is no reason to define it unless you want them to be different.

Reply:
For java the options is passed directly to java as far as i can see in the code.

Check doc for javac

https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javac.html#crosscomp-options

Reply:
It appears to differ from java version to java version. So i guess its best to just define both.

Reply:
OK, can you put the target compatibility back in place?